week-3/A-array-find/exercise.js

```diff
@@ -20,7 +22,7 @@ var names = [
   "Ahmed"
 ];
 
-var longNameThatStartsWithA = findLongNameThatStartsWithA(names);
```

Comment:
Good. The idea here is that you leave the original call to the function `findLongNameThatStartsWithA(names)` and write the function itself using `find`. If you do it this way, then the parameter passed to `findLongNameThatStartsWithA` is no longer a list, it is a single string. Neither the function name `findLongNameThatStartsWithA` or the parameter `names` then really correspond to how they are being used.